ome-dundeeomero.yml

```diff
@@ -245,29 +256,38 @@
     #omero_server_db_dumpdir_parent: /tmp
     #omero_server_db_dumpdir_name: nightly-pg_dump_omero.dir
 
-    ice_version: "3.6"
+    nginx_version: 1.16.1
     postgresql_version: "9.6"
     filesystem: "xfs"
-    omero_figure_release: "{{ omero_figure_release_override | default('v4.0.2') }}"
-    omero_server_config_set:
+    omero_figure_release: "{{ omero_figure_release_override | default('4.2.0') }}"
+
+    omero_web_config_set_production:
       omero.db.poolsize: 60
       omero.jvmcfg.percent.blitz: 50
       omero.jvmcfg.percent.indexer: 20
       omero.jvmcfg.percent.pixeldata: 20
       omero.jvmcfg.system_memory: 17000
-      omero.ldap.base: "{{ omero_server_ldap_base }}"
+      omero.ldap.base: "{{ omero_server_ldap_base | default('example') }}"
       omero.ldap.config: true
-      omero.ldap.urls: "{{ omero_server_ldap_urls }}"
+      omero.ldap.urls: "{{ omero_server_ldap_urls | default('ldap://example.org') }}"
       omero.mail.config: true
-      omero.mail.from: "{{ omero_server_mail_from }}"
-      omero.mail.host: "{{ omero_server_mail_host }}"
+      omero.mail.from: "{{ omero_server_mail_from | default('omero@example.org') }}"
+      omero.mail.host: "{{ omero_server_mail_host | default('smtp.example.org') }}"
       omero.ldap.new_user_group: "My Data"
       omero.search.batch: 100
       omero.security.password_provider: chainedPasswordProvider431
       omero.throttling.method_time.error: 60000
-      omero.Ice.Default.Host: "{{ omero_server_ice_default_host }}"
-      Ice.Admin.Endpoints: "{{ omero_server_ice_admin_endpoints }}"
-      omero.data.dir: "{{ omero_server_datadir  }}"
+      omero.Ice.Default.Host: "{{ omero_server_ice_default_host | default('127.0.0.1') }}"
+      Ice.Admin.Endpoints: "{{ omero_server_ice_admin_endpoints | default('tcp -h 127.0.0.1') }}"
+      omero.data.dir: "{{ omero_server_datadir | default('/OMERO') }}"
+
+    # Production config can't be tested in molecule
+    omero_server_config_set: "{{ molecule_test | default(False) | ternary({}, omero_web_config_set_production) }}"
```

Comment:
The propagation of `molecule_test` is a bit worrisome.

Reply:
I tried using tags without success. If you get it working let me know!